### PR TITLE
Add Zilliqa address / config

### PR DESCRIPTION
**Pull request by armatusmiles**
## Checklist
- [x] [Implement](https://github.com/trezor/trezor-firmware/pull/42) compression coords for ecdsa in (wallet-core) trezor lib
- [x] [Implement](https://github.com/trezor/trezor-firmware/pull/93) Schnorr signature in (wallet-core) trezor lib
## Blockchain checklist

- [ ] Implement functionality in C++. Put it in a subfolder of `src/`.
    - [x] Address
    - [ ] Transaction 
    - [ ] Signer
- [x] Write unit tests. Put them in a subfolder of `tests/`.
    - [x] `Mnemonic phrase - > Address` derivation test. Put this test in the `CoinTests.cpp` file
- [x] Add relevant constants in `TWCoinType`, `TWP2SHPrefix`, `TWEthereymChainID`, `TWHRP`, etc., as necessary.
- [x] Return correct curve and purpose in `src/Coin.cpp`.
- [x] Implement address validation and derivation in `src/Coin.cpp`.
- [x] Implement coin configuration `src/include/TWCoinTypeConfiguration.cpp`.
- [ ] Write interface header in `include/TrustWalletCore` and implement the interface in `src/interface`.
    - [x] Address interface (if necessary).
    - [ ] Signing interface.
- [ ] Validate generated code in Android an iOS projects. Write integration tests for each.
- [ ] Add a coin icon to "Supported Blockchains" section in `README.md`
    - [ ] Upload coin icon to [TrustWallet/tokens](https://github.com/TrustWallet/tokens) if necessary

### CODE REVIEW

**Comment by armatusmiles** [comment]
My attempts realization schnorr on trezor-based bignums [here](https://github.com/armatusmiles/trezor-crypto/blob/0cb7c2b383586ec5a0e2654e52290722472fe1f4/tests/test_check.c#L8420). Failed last test.
Test cases i got from [zilliqa code](https://github.com/Zilliqa/Zilliqa/blob/master/src/libCrypto/Schnorr.cpp#L88)

For run: 
```
make
./tests/test_check
``` 

[Describe](https://github.com/trezor/trezor-firmware/issues/88) main problem

**Comment by evesnow91** [comment]
@armatusmiles https://github.com/trezor/trezor-firmware/issues/88#issuecomment-485000929

**Comment by armatusmiles** [comment]
> @armatusmiles [trezor/trezor-firmware#88 (comment)](https://github.com/trezor/trezor-firmware/issues/88#issuecomment-485000929)

thanks

File: src/interface/TWZilliqaAddress.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    return new TWZilliqaAddress{ Address(publicKey->impl) };
+}
+
+void TWZilliqaAddressDelete(struct TWZilliqaAddress *_Nonnull address) {
+    delete address;
+}
+
+TWString *_Nonnull TWZilliqaAddressDescription(struct TWZilliqaAddress *_Nonnull address) {
+    const auto string = address->impl.string();
+    return TWStringCreateWithUTF8Bytes(string.c_str());
```
Comment (vikmeup) [inline src/interface/TWZilliqaAddress.cpp]: is there a way in C++ to call this `TWStringCreateWithUTF8Bytes(address->impl.string());` ?

Comment (hewigovens) [reply src/interface/TWZilliqaAddress.cpp]: this is C interface, we need to return `TWString`

**Pull request merged**